frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -249,6 +256,19 @@ export const funnelLogic = kea<funnelLogicType>({
             actions.setConversionWindowInDays(days)
             actions.loadResults()
         },
+        openPersonsModal: ({ step, stepNumber }) => {
+            trendsLogic().actions.setShowingPeople(true)
+            trendsLogic().actions.loadPeople(
+                { id: step.action_id, name: step.name, properties: [], type: step.type },
+                `Persons who completed Step #${stepNumber} - "${step.name}"`,
+                '',
+                '',
+                '',
+                true,
+                '',
+                stepNumber
+            )
```

Comment:
A nit for now and outside the scope of the PR, but this `loadPeople` could get a refactor into basically `loadPeople(action, options)`. Having that many `'',` in a row feels like something could be improved :).

Reply:
yeah, I've kind of been able to connect the funnel/trends logics together so I think I'll try to refactor this into the personsModalLogic as originally planned, in a separate PR 😃